engine/Dockerfile

```diff
@@ -19,9 +19,12 @@ RUN pip install -r requirements.txt
 # https://stackoverflow.com/questions/34398632/docker-how-to-run-pip-requirements-txt-only-if-there-was-a-change/34399661#34399661
 COPY ./ ./
 
-# Collect static files and create an SQLite database
-RUN mkdir -p /var/lib/oncall
+# Collect static files
 RUN DJANGO_SETTINGS_MODULE=settings.prod_without_db DATABASE_TYPE=sqlite3 DATABASE_NAME=/var/lib/oncall/oncall.db SECRET_KEY="ThEmUsTSecretKEYforBUILDstage123" SILK_PROFILER_ENABLED="True" python manage.py collectstatic --no-input
+
+# Create SQLite database and set permissions
+RUN mkdir -p /var/lib/oncall
+RUN DATABASE_TYPE=sqlite3 DATABASE_NAME=/var/lib/oncall/oncall.db python manage.py create_sqlite_db
```

Comment:
Adding a new `manage.py` command that specifically creates a DB file so it's present in the docker image before setting permissions